src/views/Rankings.vue

```diff
@@ -41,20 +48,28 @@ export default {
       leaderboards: 'leaderboards/leaderboards',
     }),
   },
-
   watch: {
     currentLeaderboard(leaderboard) {
       const index = this.leaderboards.findIndex(l => l.id === leaderboard.id)
       this.$refs.snapNav.goToPage(index)
     },
   },
-
   methods: {
     ...mapActions({
       selectLeaderboard: 'leaderboards/selectLeaderboard',
       fetchLeaderboards: 'leaderboards/fetchLeaderboards',
       joinLeaderboard: 'leaderboards/joinLeaderboard',
     }),
+    handleChangeItem(index) {
+      if (this.currentIndex == index) {
```

Comment:
Its usually better to use 3 ='s for stricter comparisons (type)

Reply:
ruby 🤦